app/Models/FAQ.php

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App\Models;
+
+use Illuminate\Database\Eloquent\Factories\HasFactory;
+use Illuminate\Database\Eloquent\Model;
+
+class FAQ extends Model
+{
+    use HasFactory;
+
+    protected $table = 'faqs';
```

Comment:
The table name is not required if you follow Laravel conventions. It might be that the class name you chose made Laravel to create a table name like f_a_qs. That is because of the all capitals. If you chose Faq as model name, Laravel would genereate the proper table name